code-challenges.test.js

```diff
@@ -69,5 +108,23 @@ const accountTransactions3 = []
 // Expected output: []
 
 // b) Create the function that makes the test pass.
+const sumIt = (input) => {
+  let arr = []
+  arr.push(input[0])
+  for(let i = 1; i < input.length ; i++) {
+    console.log(arr);
+    arr.push(input[i] + arr[i - 1])
+  }
+  
+  return arr
+}
 
-// Pseudo code:
+  sumIt(accountTransactions1)
+  sumIt(accountTransactions2)
+  sumIt(accountTransactions3)
```

Comment:
I would remove these function calls for all of the questions, as you are already performing them in your tests.